modules/monitoring/files/usr/lib/nagios/plugins/check_reboots_required

```diff
@@ -1,10 +1,34 @@
 #!/usr/bin/env ruby
 
+def create_machines_hash
+  machines_list= `/usr/local/bin/govuk_node_list --list`.split("\n")
+  machines_names_hash = Hash.new
+
+  machines_list.each do |machine|
+    machine_fqn_name = machine.split(":")
+    machines_names_hash[machine_fqn_name[0]]=machine_fqn_name[1]
+  end
+
+  return machines_names_hash
+end
+
+def hydrate_hostname(hostname, machines_names_hash)
+  name=machines_names_hash[hostname]
+
+  if (name != nil)
+    return "%s with hostname: %s" % [name, hostname]
+  end
+
+  return hostname
```

Comment:
`return` is implicit in Ruby for the last expression in a function, so you could replace the above few lines by this I think.

```ruby
if name
  "%s with hostname: %s" % [name, hostname]
else
  hostname
end
```